cmake/CPackConfig.cmake

```diff
@@ -108,7 +108,7 @@ if(UNIX AND NOT APPLE)
   set(CPACK_COMPONENTS_GROUPING ALL_COMPONENTS_IN_ONE)
   set(CPACK_DEB_COMPONENT_INSTALL "ON")
   set(CPACK_DEBIAN_PACKAGE_SHLIBDEPS "ON")
-  set(CPACK_DEBIAN_PACKAGE_DEPENDS "fonts-linuxlibertine (>= 5.3.0)")
+  set(CPACK_DEBIAN_PACKAGE_DEPENDS "fonts-linuxlibertine (>= 5.3.0), lua (>= 5.3.0)")
```

Comment:
Our package doesn't depend on the Lua interpreter. If installing it to silence `luarocks` warnings, just have the user install it alongside `luarocks` in the instructions.